code/controllers/subsystems/virtual_reality.dm

```diff
@@ -230,7 +249,7 @@ SUBSYSTEM_DEF(virtualreality)
 
 	mind_transfer(user, robot[choice])
 
-/datum/controller/subsystem/virtualreality/proc/bound_selection(var/user, var/network)
+/datum/controller/subsystem/virtualreality/proc/bound_selection(var/user, var/network, var/return_list)
```

Comment:
`return_list` seems useless here, you can always return the list as it doesn't seem you return anything else